### ISSUE TITLE
Keyboard-specific interaction lag when working with VS code and assistive technology screenreader

### ISSUE THREAD

**Issue by jacob-kruger-work** [microsoft/vscode#143261]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version:1.64.2
- OS Version:windows 10 64 bit 21H2

Steps to Reproduce:
Up until recently, was quite comfortably working with VS code, using the NVDA screenreader form of assistive technology since I am 100% blind.
Am specifically working with python and django web development framework, but, don't think that part of it is too relevant, since this issue persists if I try working on something like a single python source code file, focusing on something different.
Main thing is that, recently, I started experiencing a bit too much feedback and reaction to keyboard input lag that is actually almost seeming to block some keystrokes, which is then messing up source code, etc., and, had been quite comfortably using VS code as my primary development environment for more than 6 months now.
Maybe this is to do with some or other recent version update and, it was honestly quite fine up until roundabout a week ago, so, maybe I should try rolling back to a previous version?
Did also try out some other alternatives in terms of screenreader - tried it with both MS narrator and Jaws, since am operating under windows 10 64 bit, but, pretty much the same issue, and, not relevant to rest of OS interaction.
Also posted this on stack overflow in case:
[Stack overflow post - Keyboard-specific interaction lag when working with VS code and assistive technology screenreader](https://stackoverflow.com/questions/71153632/keyboard-specific-interaction-lag-when-working-with-vs-code-and-assistive-techno)

**Comment by LittleStar-VIP** [comment]
Hi, I am a nvda user, and is also experiencing significant lag in using vs code to edit python script.
I am using the same OS version and VS Code version as the original post.
Say for example, I am editing a few line, (3 line of python codes) I will not experience lag.
But say for example I am editing a python file containing 3xk words 
if I try to move from word to word by ctrl left or right arrow, it takes more than an second before nvda can anounce something.

**Comment by jacob-kruger-work** [comment]
Update - if roll back to 1.63.2 from November 2021, it is again somewhat better, if not necessarily perfect, but, definitely better and more usable:
[Visual Studio Code November 2021](https://code.visualstudio.com/updates/v1_63)

**Comment by hediet** [comment]
Unfortunately, I cannot reproduce this with VS Code 1.64.2 and nvda.

**Comment by hediet** [comment]
@isidorn any ideas?

**Comment by isidorn** [comment]
This might be related to the Electron 16 update which we did in VS Code 1.64.2
@jacob-kruger-work can you reproduce if you open https://vscode.dev/github/microsoft/vscode in your Chrome browser and try to create some python file?
If that reproduces, do you reproduce if you type in this textArea here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

**Comment by jacob-kruger-work** [comment]
Sorry - had clicked on close with comment button

**Comment by isidorn** [comment]
@jacob-kruger-work based on what you are saying it sounds like a Chrome textArea issue - since you also reproduce it just with Chrome. Would you mind filling a new issue for Chromium and I can ping some Googlers to investitage
https://bugs.chromium.org/p/chromium/issues/list

**Comment by jacob-kruger-work** [comment]
New issue posted: https://bugs.chromium.org/p/chromium/issues/detail?id=1298980

**Comment by isidorn** [comment]
@jacob-kruger-work thanks! I have commented on the Chromium issue. Let's see what they say.

**Comment by isidorn** [comment]
@jacob-kruger-work can you still reproduce this issue with latest VS Code?

**Comment by jacob-kruger-work** [comment]
Haven't noticed this issue occurring for quite some time now

**Comment by isidorn** [comment]
Awesome, thanks!
